tools/ports/bullet.py

```diff
@@ -0,0 +1,48 @@
+import os, shutil, logging, subprocess, sys, stat
+
+TAG = 'version_1'
+
+def build_with_configure(ports, shared, path):
+  if not sys.platform.startswith('win'): #TODO: test on windows
```

Comment:
this definitely will not work on windows, i am afraid. we need to avoid scripts like autogen, configure, etc.

often they just emit a `config.h` file with the detected int type, whether libc supports some features, etc, and we can just include that file in the repo. If the output contains absolute paths though, it might need more work.